indico/modules/events/abstracts/controllers/management.py

```diff
@@ -103,11 +104,39 @@ def _process(self):
             contrib_session = abstract_data.pop('contrib_session', None)
             merge_persons = abstract_data.pop('merge_persons', None)
             judge_abstract(self.abstract, abstract_data, judgment=judgment, contrib_session=contrib_session,
-                           merge_persons=merge_persons, send_notification=send_notification, )
+                           merge_persons=merge_persons, send_notification=send_notification)
         return WPManageAbstracts.render_template('management/abstract.html', self._conf, abstract=self.abstract,
                                                  form=form)
 
 
+class RHManageBulkAbstractJudgment(RHManageAbstractsActionsBase):
+    """Perform bulk judgment operations on selected abstracts"""
+
+    def _process(self):
+        abstract_ids = request.form.getlist('abstract_id')
+        form = AbstractsJudgmentForm(event=self.event_new, abstract_id=[a.id for a in self.abstracts])
+        form.judgment.data = request.values.get('action')
+        if form.process_ajax():
+            return form.ajax_response
+        if form.validate_on_submit():
+            abstract_data = form.data
```

Comment:
maybe a split_data property returning a tuple with the special values and the remaining data dict?